Sources/Sentry/SentryProfiler.mm

```diff
@@ -146,6 +162,12 @@ - (void)start
         sampledProfile[@"queue_metadata"] = queueMetadata;
         _profile[@"sampled_profile"] = sampledProfile;
         _startTimestamp = getAbsoluteTime();
+        _startDate = [NSDate date];
+
+        [SentryLog
+            logWithMessage:[NSString stringWithFormat:@"Starting profiler at system time %llu.",
```

Comment:
`l`: We could also add the _startDate to the log message.

Reply:
Gonna skip this for now, leaving it for if it's helpful to someone later. Don't want to incur the date formatter overhead for logging from profiles.